assets/models/system/automl-text-classification/spec.yaml

```diff
@@ -0,0 +1,51 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: AutoML-Text-Classification
+path: ./
+properties:
+  finetune-min-sku-spec: 4|1|28|176
+  finetune-recommended-sku: Standard_NC12s_v3
+  finetuning-tasks: text-classification
```

Comment:
can you also please fix this for image segmentation task?

Reply:
fixed :)